charts/pulsar/templates/bookkeeper/bookkeeper-autorecovery-statefulset.yaml

```diff
@@ -70,6 +70,10 @@ spec:
 {{ toYaml . | indent 8 }}
 {{- end }}
     {{- end }}
+      {{- if and .Values.affinity.anti_affinity .Values.autorecovery.custom_affinity}}
```

Comment:
Overall looks good. The flags used here seems a bit confusing when I took the first look at them.

Reply:
yes... I agree, but it is the pattern already used so we are a bit locked in here :\